### PR TITLE
(WIP Docs) Added tuning params to Prometheus remote write section 

### PR DESCRIPTION
**Pull request by gibbscullen**
**What this PR does / why we need it**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

File: docs/coordinator/api/remote.md [hunk 1]
```diff
@@ -30,6 +30,33 @@ docs/common/headers_optional_read_write.md
 
 Binary [snappy compressed](http://google.github.io/snappy/) Prometheus [WriteRequest protobuf message](https://github.com/prometheus/prometheus/blob/10444e8b1dc69ffcddab93f09ba8dfa6a4a2fddb/prompb/remote.proto#L22-L24).
 
+### Available Tuning Params 
+
+**Note:** All relevant parameters can be found under the `queue_config` section of the Prometheus `remote_write` configuration.
```
Comment (benraskin92) [inline docs/coordinator/api/remote.md]: I would just link to the Prometheus website so that this doesn't go out of date?

Comment (gibbscullen) [reply docs/coordinator/api/remote.md]: Like just link, or add the link to what I already have?

Comment (benraskin92) [reply docs/coordinator/api/remote.md]: I would just have the link otherwise what you have might diverge at some point.

Comment (gibbscullen) [reply docs/coordinator/api/remote.md]: Link added.

**Pull request merged**